### DISCUSSION TITLE
Any reason why use cu116 instead of automatic default 113?

### DISCUSSION THREAD

**Discussion by mhnoni** [d8ahazard/sd_dreambooth_extension#157] [General]
And is there any easy way to manually install cu116 since I have slow internet? I downloaded `torch-1.12.1+cu116-cp310-cp310-win_amd64.whl` but when I try to install it using pip in venv env, I get this 
`ERROR: torch-1.12.1+cu116-cp310-cp310-win_amd64_2.whl is not a supported wheel on this platform.`

**Comment by sgsdxzy** [comment]
bitsandbytes and thus AdamW8bit uses platform-specific binaries, and as Windows is not officially supported we all use the prebuilt windows binaries from a third party repo, which only includes 116.
If you use Linux you can use 113.

**Reply by mhnoni** [reply]
> bitsandbytes and thus AdamW8bit uses platform-specific binaries, and as Windows is not officially supported we all use the prebuilt windows binaries from a third party repo, which only includes 116. If you use Linux you can use 113.

Thanks for the info, can't you compile it for 113 for windows? 
And what is the best way to upgrade automatic to torch-1.12.1+cu116? I used pip install but that gave me the error I mentioned.
